tests/Behat/Extension/UniqueHtmlEditorConfigExtension.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace DNADesign\Elemental\Tests\Behat\Extension;
+
+use DNADesign\Elemental\Models\BaseElement;
+use SilverStripe\Core\Extension;
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\HTMLEditor\HTMLEditorConfig;
+use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
+
+/**
+ * @extends Extension<BaseElement>
```

Comment:
This adds no benefit but does no harm in 4.x.
In 5.x it tells your IDE what `$this->owner` and `$this->getOwner()` are.